app/models/english_proficiency.rb

```diff
@@ -1,4 +1,6 @@
 class EnglishProficiency < ApplicationRecord
+  include PublishedInAPI
```

Comment:
Included this here (rather than in the actual EFL models) for multiple reasons:
- This cannot be included in the EFL models, since it requires the application form to be always available. However, you can have a standalone EFL qualification object (with no associated application_form when it's created). To make this work, then we'd have to enforce weird rules about the order of creation of `english_proficiency` and `efl_qualifications`.
- This model also affects the API response for an application choice. It decides on the value of `english_main_language`

This works as intended because the touch from the EFL qualifications triggers the after_commit on the english_proficiency and hence the touch on the application choices